src/box/sql/where.c

```diff
@@ -4606,10 +4606,10 @@ sqlWhereBegin(Parse * pParse,	/* The parser context */
 					assert(wctrlFlags &
 					       WHERE_ONEPASS_DESIRED);
 					while (pJ->def->iid != idx_def->iid) {
-						iIndexCur++;
 						iid++;
 						pJ = pTabItem->space->index[iid];
 					}
+					iIndexCur++;
```

Comment:
Why third? Why no crash with a second index? I can't understand it from the code. Tbh, I understand this code at all :) - like, what is `iIndexCur`? What is `iAuxArg`? But I suppose it should be fine if the tests pass.